raiden-network/light-client
# Uses locally hosted version of icons and fonts

Fixes #1498  

**Short description**
Introduces packages for used icons and fonts.

**Definition of Done**


**Steps to manually test the change (dApp)**

1. Connect to the page
2. Open console
3. Reload page
4. Verify no call to googlefonts is performed.
